apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -184,7 +184,7 @@ public function enqueue_assets() {
 				'templates'    => array_merge( $default_templates, $page_templates ),
 				// phpcs:ignore WordPress.Security.NonceVerification.Recommended
 				'screenAction' => isset( $_GET['new-homepage'] ) ? 'add' : $screen->action,
-				'theme'        => normalize_theme_slug( get_stylesheet() ),
+				'stylesheet'   => get_stylesheet(),
```

Comment:
Not sure why the returned value of `get_stylesheet()` doesn't have `repo` prefix such as `pub/`, `premium`, etc. Prepare to revert this change: https://github.com/Automattic/wp-calypso/pull/69553